content/en/docs/architecture/timeouts.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Timeouts of a Testrun"
+date: 2020-12-21T23:27:39-04:00
+draft: false
+---
+
+We have timeouts for different aspects of a testrun. Some of them are applied to all jobs while the others are configurable in jobs' definition.
+
+# Prow job timeout
+
+`4h`: [the amount of time before Prow sends `SIGINT` to the job](https://github.com/kubernetes/test-infra/blob/6d5079cd601273d85d9c57a6eccdfedc853d8376/prow/apis/prowjobs/v1/types.go#L330). It is configured in [Prow's config](https://github.com/openshift/release/blob/9238ee8b89c861793f9b8e8f7f6509abe33bd0b8/core-services/prow/02_config/_config.yaml#L559) for all jobs.
```

Comment:
Prow does not send SIGINT. We delete the Pod, and the `kubelet` does this.

Reply:
Updated.